Formula/haskell-language-server.rb

```diff
@@ -21,10 +21,14 @@ class HaskellLanguageServer < Formula
 
   depends_on "cabal-install" => [:build, :test]
   depends_on "ghc" => [:build, :test]
+  depends_on "haskell-language-server-wrapper"
 
   def install
     system "cabal", "v2-update"
     system "cabal", "v2-install", *std_cabal_v2_args
+    ghc_version = Formula["ghc"].version.major_minor
+    bin.install bin/"haskell-language-server" => "haskell-language-server-#{ghc_version}"
+    rm bin/"haskell-language-server-wrapper"
```

Comment:
Are you sure you still need this line? This should've been removed by the `bin.install`, I think,

Reply:
`bin.install` concerns the `haskell-language-server` and this `rm`s the `haskell-language-server-wrapper`, right?

Reply:
Ah yes, misread, sorry.